### ISSUE TITLE
Update PDF to contain a list of past versions and the suggested citation

### ISSUE THREAD

**Issue by janineA** [NCAR/aircraft_ProcessingAlgorithms#4]
Page 2 of the PDF is currently blank. Update it to contain a list of all previous versions as well as a prominent suggested citation that includes the DOI. This citation should match the citation in the README.md file.

**Comment by janineA** [comment]
"lyx" must be run on mercury. barolo and tikal give an error that the version of lyx is too old.

I ran "lyx ProcessingAlgorithms.lyx" as suggested in step 1 of the instructions at the bottom of the PDF. It printed a bunch of errors like:

> svn: E200009: Could not display info for all targets because some targets don't exist
> Systemcall.cpp (276): Systemcall: 'svn info "ProcessingAlgorithms.lyx"' finished with exit code 1
> svn: warning: W155010: The node '/net/opt_lnx/local_el7_64/janine/aircraft_ProcessingAlgorithms/Section1.lyx' was not found.> 

This code is (clearly) not in GitHub so this functionality needs to update. Since I manually did a git pull right before my lyx command, I chose to ignore these for now and press on.

The files opens and seems fine (not having ever used lyx before, I may be missing something.). As a null activity I decided to do a file-> export -> pdflatex without making any changes to confirm it worked. 

I got an error that Date is used in the wrong place in the layout, which I ignored for this test.
I then got two fatal errors. Given that I have higher priority items at the moment, I am setting this aside for now.

![Screen Shot 2021-12-17 at 1 10 29 PM](https://user-images.githubusercontent.com/4854954/146602164-aee62766-89b4-42c2-b9b1-633a85e9a1cf.png)
